crates/admin/src/rest_api/invocations.rs

```diff
@@ -116,3 +118,145 @@ pub async fn delete_invocation<V, IC>(
         Ok(StatusCode::ACCEPTED)
     }
 }
+
+generate_meta_api_error!(KillInvocationError: [InvocationNotFoundError, InvocationClientError, InvalidFieldError, InvocationWasAlreadyCompletedError]);
```

Comment:
I am not sure whether we are optimizing here for the reader or the writer of the code.